src/marvin/settings.py

```diff
@@ -87,6 +87,7 @@ class Settings(MarvinBaseSettings):
     )
     llm_temperature: float = 0.8
     llm_request_timeout_seconds: Union[float, list[float]] = 600.0
+    format_response_retries: int = 0
```

Comment:
This is called "format response" but is being used to configure every call I think

Reply:
good point - maybe `function_call_retries`? but im not sure if i like that because its kind of vague - any suggestions?